apps/frontend/package.json

```diff
@@ -43,18 +43,18 @@
     "@types/luxon": "^3.3.1",
     "@types/mustache": "^4.1.1",
     "@types/node": "^20.1.0",
-    "@types/objects-to-csv": "^1.3.0",
     "@types/prismjs": "^1.16.1",
     "@types/sanitize-html": "^2.3.1",
     "@types/triple-beam": "^1.3.2",
     "@types/uuid": "^9.0.0",
     "@types/vuelidate": "^0.7.13",
     "@types/xml-js": "^1.0.0",
-    "@vue/cli": "^5.0.3",
+    "@types/yazl": "^2.4.2",
```

Comment:
yazl does not seem to be maintained any longer.  we should look into a replacement lib to handle zip files

but also why do we need it at all when we also apparently are using adm-zip which is also a zip library?

admittedly it seems like adm-zip is also probably not being maintained